src/styles/abstract/_mixins.scss

```diff
@@ -0,0 +1,29 @@
+@mixin font($size, $weight: 400, $height, $style: normal) {
+  font-size: $size;
+  line-height: $height;
+  font-weight: $weight;
+  font-style: $style;
+}
+
+@mixin border($size, $color, $raduis: none) {
```

Comment:
There is a small typo in the parameter name, it should be `$radius` instead of `$raduis`. This can lead to confusion in the future when using this mixin.